### PR TITLE
Add Sentry Spotlight

### PR DESCRIPTION
**Pull request by ixartz**

### CODE REVIEW

File: sentry.client.config.ts [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+// The config you add here will be used whenever a users loads a page in their browser.
+// https://docs.sentry.io/platforms/javascript/guides/nextjs/
+
+import * as Sentry from '@sentry/nextjs';
+import * as Spotlight from '@spotlightjs/spotlight';
+
+Sentry.init({
+  // dsn: '___DSN___',
+
+  spotlight: process.env.NODE_ENV === 'development',
```
Comment (Shubhdeep12) [inline sentry.client.config.ts:11]: I dont think its mandatory to add this boolean for nextjs

Comment (Shubhdeep12) [reply sentry.client.config.ts:11]: For client config

Comment (ixartz) [reply sentry.client.config.ts]: Thank you for your feedback, I remove the spotlight attribute in Sentry client side configuration.

Indeed, it's not needed for client config.

File: package.json [hunk 2]
```diff
@@ -29,6 +31,8 @@
     "@clerk/nextjs": "^4.27.5",
     "@hookform/resolvers": "^3.3.2",
     "@libsql/client": "^0.3.6",
+    "@sentry/nextjs": "^7.86.0",
+    "@spotlightjs/spotlight": "^1.1.1",
```
Comment (HazAT) [inline package.json:35]:
```suggestion
    "@spotlightjs/spotlight": "^1.2.2",
```

File: package.json [hunk 3]
```diff
@@ -51,6 +55,7 @@
     "@playwright/test": "^1.40.1",
     "@semantic-release/changelog": "^6.0.3",
     "@semantic-release/git": "^10.0.1",
+    "@spotlightjs/sidecar": "^1.1.1",
```
Comment (HazAT) [inline package.json:58]:
```suggestion
```
This dep shouldn't be required, the sidecar binary is also part of spotlight package.

**Review by HazAT — CHANGES_REQUESTED** [review]

**Pull request merged**